MatrixSDK/MXSession.m

```diff
@@ -2898,6 +2893,10 @@ - (void)removeRoom:(NSString *)roomId
                                                                      kMXSessionNotificationRoomIdKey: roomId
                                                                      }];
     }
+
+    // Clean the store
+    [self.aggregations resetDataInRoom:roomId];
+    [self.store deleteRoom:roomId];
```

Comment:
Is it intentional to not call `[self.store.roomSummaryStore removeSummaryOfRoom:roomId];` anymore?
Is the room really removed from the store (and room lists)?

Reply:
Yes, it’s already called in `MXFileStore`.